plugins/woocommerce-blocks/bin/wp-env-config.sh

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
-
-## Check if user already exists
-wp user get customer 2> /dev/null
-
-## if 0 is the exit code then we can leave otherwise we'll try creating the user
-if [ $? -eq 0 ]
-then
-	EXIT_CODE=0
-else
-  wp user create customer customer@woocommercecoree2etestsuite.com --user_pass=password --role=customer
-  EXIT_CODE=$?
-fi
-
-## set permalinks for easier wp-json
-wp rewrite structure '/%postname%/' --hard
-wp core version --extra
-wp plugin list
-wp theme activate storefront
-wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}' --shipping='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}'
-## Prepare translation for the test suite
-wp language core install nl_NL
-wp language plugin install woocommerce nl_NL
-## No need to install the plugin anymore
-# wp plugin activate woocommerce-blocks
-## We download a full version of .po (that has translation for js files as well).
-curl https://translate.wordpress.org/projects/wp-plugins/woo-gutenberg-products-block/stable/nl/default/export-translations/ --output ./wp-content/languages/plugins/woo-gutenberg-products-block-nl_NL.po
-sleep 5
-## We update our po file with updated file paths.
-## This should account for when a file changes location between versions.
-php -d memory_limit=2024M "$(which wp)" i18n make-pot ./wp-content/plugins/$1/ ./wp-content/languages/plugins/woo-gutenberg-products-block-nl_NL.po --merge --domain=woo-gutenberg-products-block --exclude=node_modules,vendor --skip-audit
-## We regenerate json translation from the new po file we created.
-php -d memory_limit=2024M "$(which wp)" i18n make-json ./wp-content/languages/plugins/woo-gutenberg-products-block-nl_NL.po
-exit $EXIT_CODE
```

Comment:
Following the testing steps it complained about missing script `wp-env:config` after generating translations:
```bash
Success: Updated 'woocommerce_coming_soon' option.
✔ Ran `wp option update woocommerce_coming_soon no` in 'tests-cli'. (in 1s 146ms)
Generating test translations
npm ERR! Missing script: "wp-env:config"
```

I see this was removed, and thought the testing steps could be outdated, but when I saw this file contents I was left wandering if this part was removed by accident:

```shell
if [ $? -eq 0 ]
then
	EXIT_CODE=0
else
  wp user create customer customer@woocommercecoree2etestsuite.com --user_pass=password --role=customer
  EXIT_CODE=$?
fi

## set permalinks for easier wp-json
wp rewrite structure '/%postname%/' --hard
wp core version --extra
wp plugin list
wp theme activate storefront
wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}' --shipping='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}'
```

Can you confirm this wasn't a mistake?
Also is there any chance that removing `wp-env:config` might break any other automations we have besides the CI jobs on this PR? 🤔

Reply:
This wasn't a mistake, the testing instructions were outdated, I've updated them. But this did point to some redundant scripts that were still relying on `wp-env:config`, so I've removed those in https://github.com/woocommerce/woocommerce/pull/52188/commits/6e0c7aee2f0af647654badb98676998435e97a61

Reply:
I'm not referring to `wp-env:config` calls within our codebase, but I'll guess we'll find out on the betas and RC if any process we have eyes on breaks. But third-party agencies might have this in their pipelines, and although it is their responsibility to adjust, we probably should keep the command, just changing it into echoing a deprecated notice. And also keep the `wp-env-config.sh` doing the deprecated echo there in case someone is calling it directly.
This way we ensure we warn before breaking.

Also I'm not sure if I expressed myself correctly, to be clear, is this part redundant and safe to remove?

```bash
## set permalinks for easier wp-json
wp rewrite structure '/%postname%/' --hard
wp core version --extra
wp plugin list
wp theme activate storefront
wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}' --shipping='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}'
```